blog/serialize-a-style-using-your-chakra-ui-theme.md

```diff
@@ -0,0 +1,113 @@
+# Serialize a style using your Chakra UI theme
+
+A time ago I had to build a _subscription form_ using the [Spreedly iFrame API](https://docs.spreedly.com/reference/iframe/v1/) to allow a user to use its credit card in a safe way.
+
+After a while, I found myself into an issue with the card number and the CVV number inputs because Spreedly inserts an `iframe` and the inputs are not under my control (and my CSS).
```

Comment:
```suggestion
After a while, I found myself into an issue with the card number and the CVV number inputs because Spreedly inserts an `iframe` and the inputs are not under my control (and my CSS).
```
```suggestion
After a while, I found myself into an issue with the card number and the CVV number inputs because Spreedly inserts an `iframe`; therefore the inputs are not under my control (and my CSS).
```